packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * Deprecated. The `product-recommendation` sub-package is deprecated. Use the `commerce` sub-package instead.
```

Comment:
Nitpick, but these should be in this format:
```
@deprecated  The `product-recommendation` sub-package is deprecated. Use the `commerce` sub-package instead.
```

On 1 line instead of 2 lines. It makes it nicer to read in most IDE for implementers.

Reply:
@olamothe I intentionally split this in half to have the string show up in the documentation: https://github.com/coveo/ui-kit/pull/4088#issuecomment-2196794791

Still, I created https://coveord.atlassian.net/browse/CAPI-1124 to address this (cc @jpmarceau, @spiracha-coveo , feel free to transfert this to your board if you think it's more appropriate!)